Tests/Core/Extensions/ErrorResponse+LoggingTests.swift

```diff
@@ -0,0 +1,24 @@
+import CheckoutEventLoggerKit
+import XCTest
+
+@testable import Frames
+
+final class ErrorResponse_LoggingTests: XCTestCase {
```

Comment:
This naming convention for testing extensions looks weird. Not sure if this is a standard way of doing this

Reply:
I'm not aware of any conventions for this, so I'd be happy to discuss further. I've seen a similar style used in Apple's [Swift Protobuf](https://github.com/apple/swift-protobuf) though.

Reply:
We aren't currently following any standards for this scenario it seems. The mix of PascalCase and snake_case looks weird but happy to adopt it if it's a common approach

Reply:
We could adopt this for `IPAddressProviderNetworkTests` (`IPAddressProvider_NetworkTests`) and `ChallengeResponsePropertyValidatorTests` (`ChallengeResponse_PropertyValidatorTests`)